docs/book/src/multiversion-tutorial/deployment.md

```diff
@@ -84,8 +84,40 @@ respectively.  Notice that each has a different API version.
 Finally, if we wait a bit, we should notice that our CronJob continues to
 reconcile, even though our controller is written against our v1 API version.
 
+<aside class="note">
+
+<h1>kubectl and Preferred Versions</h1>
+
+When we access our API types from Go code, we ask for a specific version
+by using that version's Go type (e.g. `batchv2.CronJob`).
+
+You might've noticed that the above invocations of kubectl looked
```

Comment:
I wonder if this would catch non-native English readers off guard. Just not the typical contraction I am used to seeing.

Reply:
Not sure if I was trying to make this sound less nitpicky than it is or not... :man_shrugging:

Reply:
Not sure if I was trying to make this sound less nitpicky than it is or not... :man_shrugging:

Reply:
I am non-native English and IMO it is very OK. 

Just to share my IHMO: What is hard to understand for me (non-native English) is when the text does not pass the info in a straightway or more informal way to write. Usually when native speakers try to make easier is harder. :-P since when you folks are kids you learn more informal English and use too much "phrasal verbs" and when we are learning English as an outsider we see more the formal English and the phrasal verbs are crazy because they are 2/3 words/verbs that together have a completed different mean.